content/kubermatic/master/architecture/supported-providers/kubevirt/kubevirt.en.md

```diff
@@ -128,17 +134,27 @@ so all resources for this cluster are located in the `cluster-gff5gnxc7r` namesp
 It is possible to control how the tenant nodes are scheduled on the infrastructure nodes.
 ![Scheduling](/img/kubermatic/master/architecture/supported-providers/kubevirt/Scheduling.png)
 
-We provide 3 different types of scheduling for the KubeVirt tenant nodes:
-- Ensure co-location on the same infrastructure node (*Pod Affinity Preset*).
-- Prevent co-location on the same infrastructure node (*Pod Anti Affinity Preset*).
+We provide control of the user cluster nodes scheduling over topology spread constraints and node affinity presets mechanisms. You can use a combination of them, or they can work independently:
+- Spread across a given topology domains (*TopologySpreadConstraints*).
 - Schedule on nodes having some specific labels (*Node Affinity Preset*).
 
-This setup is done in the `Initial Nodes` step of KKP dashboard when creating a cluster.
+**Note**: `Pod Affinity Preset` and `Pod Anti Affinity Preset` are deprecated. Migration to `TopologySpreadConstraints` does not affect existing MachineDeployment and corresponding VMs.
+If existing MachineDeployment has Pod Affinity/Anti-Affinity Preset spec, it will remain the same. But scaling existing MachineDeployment will create new VMs which will not have Pod Affinity/Anti-Affinity Preset spec
+, instead they will have default topology spread constraint.
```

Comment:
I did not find the content of the default topology spread constraint in the doc. Is it described?

Reply:
yes, its described. Also attached is a link to Kubernetes topology constraint docs for reference.